.devcontainer/Dockerfile

```diff
@@ -13,6 +13,90 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM rapidsai/devcontainers:23.12-cpp-mambaforge-ubuntu22.04 AS base
+ARG BASE
+ARG PYTHON_PACKAGE_MANAGER
 
-ENV PATH="${PATH}:/workspaces/morpheus/.devcontainer/bin"
+FROM ${BASE} as base
+
+# ===== install common packages ==================================================================
+
+RUN <<EOF
+
+set -x
+
+apt-get update;
+
+# apt-get install -y --no-install-recommends  \
+#     libboost-{regex,system,filesystem}-dev  \
+#     ;
+
+rm -rf /tmp/*;
+rm -rf /var/tmp/*;
+rm -rf /var/cache/apt/*;
+rm -rf /var/lib/apt/lists/*;
+
+EOF
+
+# ===== install node + camouflage ================================================================
+
+COPY --from=node /usr/local/bin/node /usr/local/bin/node
+COPY --from=node /usr/local/include/node /usr/local/include/node
+COPY --from=node /usr/local/lib/node_modules /usr/local/lib/node_modules
+COPY --from=node /opt/yarn-v*/bin/* /usr/local/bin/
+COPY --from=node /opt/yarn-v*/lib/* /usr/local/lib/
```

Comment:
Is `node` a stage in the docker build?

Reply:
We probably want to mention a specific versino or make it a build variable